### PR TITLE
Remove occurences of old github org

### PR DESCRIPTION
**Pull request by bkabrda**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ node('docker') {
         dockerCleanup()
         // hack
         dir('worker') {
```
Comment (samuzzal-choudhury) [inline Jenkinsfile]: @bkabrda are we having a directory worker in the server codebase to clone fabric8-analytics-worker?

Comment (msrb) [inline Jenkinsfile]: This was necessary when the repos were private as docker couldn't clone them during build. This quick fix will enable CI/CD again. We can drop the worker hack later, it won't do any harm now :)

**Comment by msrb** [comment]
[test this]

**Comment by msrb** [comment]
@samuzzal-choudhury and we are green 😉

**Pull request merged**